packages/trip-form/i18n/en-US.yml

```diff
@@ -37,6 +37,7 @@ otpUi:
     optimizeTransfers: Fewest Transfers
     speedInMilesPerHour: "{mph} MPH" # Original units were all caps.
     walkSpeed: Walk Speed
+    walkReluctance: Walking Avoidance
```

Comment:
Hmm we need to find better text for this. How about "Minimize walking"?

Reply:
Resolved